packages/dataviz/src/formatters/formatters.ts

```diff
@@ -26,7 +26,7 @@ export function formatDateTime(value: number): string {
 export function formatNumber(value: number, precision?: number): string {
 	return value > 1e10
 		? value.toFixed(0)
-		: value.toLocaleString(i18next.language, {
+		: value.toLocaleString(i18next.language || 'en-EN', {
```

Comment:
IMHO, locale should be configured on run level

Reply:
yes need to try but from what I see i18next.init is async and even with initImmediate in test-setup file I didn't succeed to make i18next.language available :man_shrugging: